dashboard/app/models/user_level.rb

```diff
@@ -141,11 +141,10 @@ def locked=(is_locked)
     self.unlocked_at = is_locked ? nil : Time.now
   end
 
-  # this is the "locked" value we return to the client. if the lesson isn't
-  # lockable or we are an authorized teacher, we always return `false`.
+  # this is the "locked" value we return to the client.
+  # if the lesson isn't lockable, we always return `false`.
   def show_as_locked?(stage)
     return false unless stage.lockable?
-    return false if user.authorized_teacher?
```

Comment:
Where did this logic go?

Reply:
the short answer is that we don't need it. 

one thing this work has clarified is that `locked` and `disabled` aren't necessarily synonymous. we want to show the lock icon on locked levels even for authorized teachers, but we _don't want_ to disable those bubbles for authorized teachers. the disabling part is handled on the frontend, so removing this line is just to allow us to show the lock icon to authorized teachers